modules/installation-azure-service-principal.adoc

```diff
@@ -177,14 +176,16 @@ output. You need these values during {product-title} installation.
 +
 [source,terminal]
 ----
-$ az ad sp create-for-rbac --role Contributor --name <service_principal> <1>
+$ az ad sp create-for-rbac --role Contributor --name <service_principal> \ <1>
+  --scopes /subscriptions/<id>
 ----
 <1> Replace `<service_principal>` with the name to assign to the service principal.
 +
 .Example output
```

Comment:
the example output need updated based on https://github.com/openshift/openshift-docs/pull/51790/files#r1000671792